cluster/addons/storage-class/caph/default.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
```

Comment:
why do we need a StorageClass in an addons folder ?

Reply:
in this ref https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/storage-class
When we need to use external storage, one more ceph option is available.